src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs

```diff
@@ -105,8 +105,10 @@ private void ScanForComponents(MountPointScanSource source)
                         // TODO: figure out what to do with probing path registration when components are not found.
                         // They need to be registered for dependent assemblies, not just when an assembly can be loaded.
                         // We'll need to figure out how to know when that is.
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
nit: it would be good to add a comment that we keep it, however it will be internal in next version.